app/Makefile

```diff
@@ -68,7 +68,7 @@ endif
 
 APP_LOAD_PARAMS = --curve secp256k1 $(COMMON_LOAD_PARAMS) --path $(APPPATH)
 
-APP_STACK_MIN_SIZE := 3000
+APP_STACK_MIN_SIZE := 2050
```

Comment:
We need less stack size now?

Reply:
Stack was increased when we removed protobuf. I had to decrease it now to the old value because of app size constraints. 
I had a look at all the snapshots to make sure they were what we were expecting.